src/components/layout/CyVerseAppBar.js

```diff
@@ -575,160 +621,200 @@ function CyverseAppBar(props) {
-                <Drawer
-                    className={clsx(classes.drawer, {
-                        [classes.drawerOpen]: open,
-                        [classes.drawerClose]: !open,
-                    })}
-                    classes={{
-                        paper: clsx({
+                    </Drawer>
+                </Hidden>
+                <Hidden only={["sm", "md", "lg", "xl"]}>
+                    <Drawer
+                        className={clsx(classes.drawer, {
                             [classes.drawerOpen]: open,
                             [classes.drawerClose]: !open,
-                        }),
-                    }}
-                    open={open}
-                    onClose={toggleDrawer(false)}
-                >
-                    <div
-                        id={build(ids.DRAWER_MENU, ids.ACCOUNT_MI)}
-                        style={{ margin: 8 }}
+                        })}
+                        classes={{
+                            paper: clsx({
+                                [classes.drawerOpen]: open,
+                                [classes.drawerClose]: !open,
+                            }),
+                        }}
+                        open={open}
+                        onClose={toggleDrawer(false)}
                     >
-                        {accountAvatar}
-                    </div>
-                    <Divider />
-                    {drawerItems}
-                    {adminUser && (
-                        <>
-                            <Divider />
-                            {adminDrawerItems}
-                        </>
-                    )}
-                </Drawer>
-            </Hidden>
-            <CyVerseAnnouncer />
-            <main
-                className={clsx(classes.content, {
-                    [classes.contentShift]: open,
-                })}
-            >
-                {children}
-            </main>
-        </div>
+                        <div
+                            id={build(ids.DRAWER_MENU, ids.ACCOUNT_MI)}
+                            style={{ margin: 8 }}
+                        >
+                            {accountAvatar}
+                        </div>
+                        <Divider />
+                        {drawerItems}
+                        {adminUser && (
+                            <>
+                                <Divider />
+                                {adminDrawerItems}
+                            </>
+                        )}
+                    </Drawer>
+                </Hidden>
+                <CyVerseAnnouncer />
+                <main
+                    className={clsx(classes.content, {
+                        [classes.contentShift]: open,
+                    })}
+                >
+                    {children}
+                </main>
+                {newUser && (
+                    <ConfirmationDialog
+                        baseId={"de"}
```

Comment:
This id should perhaps go in an IDs file but I would give it something more descriptive like `userTourDlg` or something like that.  
Maybe the `baseId` prop is misleading and the prop should be `id` instead so it's clear it's the ID for the dialog and not like a parent ID (not for this PR, just thinking out loud).